app/models/form/sales/questions/created_by_id.rb

```diff
@@ -0,0 +1,55 @@
+class Form::Sales::Questions::CreatedById < ::Form::Question
+  ANSWER_OPTS = { "" => "Select an option" }.freeze
+
+  def initialize(id, hsh, page)
+    super
+    @id = "created_by_id"
+    @check_answer_label = "User"
+    @header = "Which user are you creating this log for?"
+    @type = "select"
+  end
+
+  def answer_options
+    return ANSWER_OPTS unless ActiveRecord::Base.connected?
+
+    User.select(:id, :name, :email).each_with_object(ANSWER_OPTS.dup) do |user, hsh|
+      hsh[user.id] = "#{user.name} (#{user.email})"
+      hsh
+    end
+  end
+
+  def displayed_answer_options(log, user = nil)
+    return ANSWER_OPTS unless log.owning_organisation
+    return ANSWER_OPTS unless user
+    return ANSWER_OPTS unless user.support? || user.data_coordinator?
+
+    users = user.support? ? log.owning_organisation.users : user.organisation.users
```

Comment:
Do we need to do this? Can it always be `log.owning_organisation.users` ? Cause data provider won't be able to see any logs other than their own org. Also, can it be assigned to someone from managing org?